docs/troubleshooting.md

```diff
@@ -5,7 +5,57 @@
 ### Why am I unable to see infrastructure logs
 Infrastructure logs are visible from the `Global` tenant and require `administrator` permissions. See the [access control](access-control.md) documentation for additional information about how a user is determined to have the `administrator` role.
 ### kube:admin is unable to see infrastructure logs
-`kube:admin` by default does not have the correct permissions to be given the admin role.   See the [access control](access-control.md) documentation for additional informations.  You may grant the permissions by:
+`kube:admin` by default does not have the correct permissions to be given the admin role.   See the [access control](access-control.md) documentation for additional information.  You may grant the permissions by:
 ```
 oc adm policy add-cluster-role-to-user cluster-admin kube:admin
-```
\ No newline at end of file
+```
+
+## Amount of logs per project
+
+The new [data model](https://github.com/openshift/enhancements/blob/master/enhancements/cluster-logging/cluster-logging-es-rollover-data-design.md#data-model) was introduced in OCP 4.5.
+Since then, the logs from individual namespaces no longer end up in dedicated indices by default, but they share a common index.
+
+To learn which projects are generating most of the logs you can use Elasticsearch query language to calculate
+aggregated statistics.
+
+The following is an example of hourly date histogram aggregation (for last three hours) with nested number of logs grouped by top namespaces:
+
+
+```json
+GET /app-*,infra-*,audit-*/_search
+{
+  "size": 0,
+  "query": {
+    "range": {
+      "@timestamp": {
+        "gte": "now-3h",
+        "lt": "now"
+      }
+    }
+  },
+  "aggs": {
+    "Histogram": {
+      "date_histogram": {
+        "field": "@timestamp",
+        "interval": "hour"
+      },
+      "aggs": {
+        "top_namespaces": {
+          "terms": {
+            "size": 10,
+            "order" : { "_count" : "desc"},
+            "field": "kubernetes.namespace_name"
+          }
+        }
+      }
+    }
+  }
+}
+```
+You can leave out any members from `[app-*,infra-*,audit-*]` list in the beginning to make the query more focused.
```

Comment:
Can we not simply use the alias in lieu of wild card matching the indices?

Reply:
+1 i think it should be `app, infra, audit` instead since we have aliases for the wildcards

Reply:
can we update the query example then and this line? it still has `(app|infra|audit)-*` in it... it should just be `(app|infra|audit)` no `-*`

Reply:
fixed